.travis.yml

```diff
@@ -0,0 +1,19 @@
+before_install:
```

Comment:
created continuous integration with travis ci

Reply:
Looks great, thanks for doing this Eric!